retriever/lib/content-security-policy.js

```diff
@@ -0,0 +1,45 @@
+// List of allowed hosts in the CSP <host-source> format:
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy#host-source
+const ALLOWED_HOSTS = [
+  'https://*.filcdn.io',
+  'https://*.calibration.filcdn.io',
```

Comment:
I believe the above string covers that one as well
```suggestion
```

Reply:
I am not sure, TBH. Let's try your proposal; we can easily add back the other entry later if needed.

Reply:
Addressed in [be2b331](https://github.com/filcdn/worker/pull/155/commits/be2b331b52f12de560d4dd679f12e7fac7fd7b03)